exporter/postgres/queries_backrest.yml

```diff
@@ -4,24 +4,38 @@
 #
 ###
 
-ccp_backrest_last_runtime:
+ccp_backrest_last_info:
     query: "WITH all_backups AS (
                 SELECT config_file
                  , jsonb_array_elements(data) AS stanza_data
                 FROM monitor.pgbackrest_info(#PGBACKREST_INFO_THROTTLE_MINUTES#)
             )
-            , per_stanza AS ( 
+            , per_stanza AS (
                 SELECT config_file
                  , stanza_data->>'name' AS stanza
                  , jsonb_array_elements(stanza_data->'backup') AS backup_data
                 FROM all_backups
             )
-            SELECT config_file
-             , stanza
-             , backup_data->>'type' AS backup_type
-             , EXTRACT( epoch FROM (max(to_timestamp((backup_data->'timestamp'->>'stop')::bigint))) - max(to_timestamp((backup_data->'timestamp'->>'start')::bigint)) ) AS backup_runtime_seconds
-            FROM per_stanza
-            GROUP BY config_file, stanza, backup_data->>'type'"
+            SELECT a.config_file
+             , a.stanza
+             , a.backup_data->>'type' AS backup_type
+             , a.backup_data->'info'->'repository'->>'delta' AS repo_backup_size_bytes
+             , a.backup_data->'info'->'repository'->>'size' AS repo_total_size_bytes
```

Comment:
Not sure on this, but I need the value returned as text, so went with the operator that just does that. Would hope it would do the same thing under the hood either way and just do the data type conversion at the end.

Reply:
Just realized you said the `#>>` operator vs the `->>` operator. For some reason I read that as `#>` which returns a json object. I'm not sure on the performance difference here, but in this instance it's really not traversing that far down and this query is only meant to run once every 10 minutes, so I don't think it makes a huge difference here. If anyone wants to do some additional testing here to find out, I'm open to changing. Otherwise, when this PR is ready to be merged, I'm just going to leave it as is.